Polish/ubranie/kieszen/otworz-kieszen.md

```diff
@@ -0,0 +1,3 @@
+Otwierasz kieszen.
+
+Okazuje sie byc bardzo gleboka. Na samym jej dnie znajdujesz maly, plastikowy gwizdek. Korci Cie zeby sprawdzic czy dziala. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.